administration/troubleshooting.md

```diff
@@ -140,6 +140,50 @@ Now we should start seeing output similar to the following:
 [2022/10/21 10:49:25] [ info] [output:null:null.0] thread worker #0 stopping...
 [2022/10/21 10:49:25] [ info] [output:null:null.0] thread worker #0 stopped
 ```
+### Parameters for the output in Tap 
+When activating Tap, any plugin parameter can be given. These can be used to modify, for example, the output format, the name of the time key, the format of the date, etc.
+
+In the next example we will use the parameter ```"format": "json"``` to demonstrate how in Tap, stdout can be shown in Json format, however this could be sent to any output plugin.
```

Comment:
however... is redundant, we just explained that in the previous paragraph.

Reply:
done